src/app.js

```diff
@@ -76,5 +77,6 @@ import './webgl/p5.RendererGL.Retained';
 import './webgl/p5.Texture';
 
 import './core/init';
+import dom from './dom/dom';
```

Comment:
Don't think this line is necessary?
```suggestion
```

Reply:
I tested this removal locally and it's not necessary.